commit 1159d9b6f2dd749adc4dbf9975bb8a27bccf2bd7 (HEAD -> master)
Author: Trent Mick <trentm@gmail.com>
Date:   2019-09-16T08:58:03-07:00 (3 weeks ago)
    
    TOOLS-2254 add redirects from mo-docs to modocs.joyent.us (restdown files should now always be .md)
    Reviewed by: Chris Burroughs <chris.burroughs@joyent.com>
    Approved by: Chris Burroughs <chris.burroughs@joyent.com>

diff --git a/Makefile b/Makefile
index bdb7a8e..88e27d6 100644
--- a/Makefile
+++ b/Makefile
@@ -16,7 +16,7 @@ TEST_FILTER ?= .*
 #
 # Files
 #
-DOC_FILES	 = index.restdown
+DOC_FILES	 = index.md
 JS_FILES	:= $(shell find lib test -name '*.js')
 JS_FILES	+= $(shell find bin -type f -not -name '.*.swp')
 JSL_CONF_NODE	 = tools/jsl.node.conf
diff --git a/docs/index.restdown b/docs/index.md
similarity index 100%
rename from docs/index.restdown
rename to docs/index.md
